Quickstart guide review (#1055)
substrate-developer-hub/substrate-docs: content/md/en/docs/quick-start/index.md

---
title: Quick start
description: Get started with Substrate
---

All of the Substrate tutorials and how-to guides require you to build and run a Substrate node in your development environment.
To this end, the [Substrate Developer Hub](https://github.com/substrate-developer-hub/) (Devhub) maintains various _templates_ for your use.

The [base node template {at the `latest` release tag}](https://github.com/substrate-developer-hub/substrate-node-template/releases/tag/latest) includes everything you need to get started with minimal features, acting as a basis to adds more to!

This _Quick start_ assumes that you are setting up a development environment for the first time and want to try out running a single template node on your local computer.

To keep things simple, you'll connect to the local node using a web browser and look up a balance for a predefined sample account.

## Before you begin

Before you begin, verify the following:

> - You have an internet connection and access to a terminal prompt on your local computer.

Author: ```suggestion
- You have an internet connection and access to a shell prompt on your local computer's terminal.
```

---

All of the Substrate tutorials and how-to guides require you to build and run a Substrate node in your development environment.
To this end, the [Substrate Developer Hub](https://github.com/substrate-developer-hub/) (Devhub) maintains various _templates_ for your use.

The [base node template {at the `latest` release tag}](https://github.com/substrate-developer-hub/substrate-node-template/releases/tag/latest) includes everything you need to get started with minimal features, acting as a basis to adds more to!

This _Quick start_ assumes that you are setting up a development environment for the first time and want to try out running a single template node on your local computer.

To keep things simple, you'll connect to the local node using a web browser and look up a balance for a predefined sample account.

## Before you begin

Before you begin, verify the following:

- You have an internet connection and access to a shell prompt on your local computer's terminal.

- You are generally familiar with software development and using command-line interfaces.

> - You have the required packages for Substrate development installed.
>   See the [Install section](/main-docs/install/) before you continue to ensure you have all you need.

Author: ```suggestion
- You have the required packages for Substrate development installed.
  **See the [Install section](/main-docs/install/) before you continue to ensure you will not run into compilation errors.**
```
